bloom/generators/debian/generator.py

```diff
@@ -575,7 +575,7 @@ def sanitize_package_name(name):
 class DebianGenerator(BloomGenerator):
     title = 'debian'
     description = "Generates debians from the catkin meta data"
-    has_run_rosdep = False
+    has_run_rosdep = bool(os.environ.get('BLOOM_SKIP_ROSDEP_UPDATE', 'False'))
```

Comment:
Maybe this doesn't do what you expect? Do you want "ifdef" like behavior or "if" like behavior?

Consider:

```
% BLOOM_SKIP_ROSDEP_UPDATE=0 python3 -c "import os; print(bool(os.environ.get('BLOOM_SKIP_ROSDEP_UPDATE', 'False')))"
True
```

Reply:
You're absolutely correct. I updated the logic, and tested both cases this time.